ui/ui-components/pages/_app.tsx

```diff
@@ -12,7 +12,7 @@ import "../components/Icons";
 
 import { WebAppContext } from "../contexts/webApp";
 
-import * as eventHandlers from "../eventHandlers";
+import "../eventHandlers";
```

Comment:
This just warm boots all those objects before so that they're instanced once the page or component uses them.